react/components/FiltersContainer.js

```diff
@@ -123,7 +123,9 @@ export default class FiltersContainer extends Component {
       hiddenFacets,
     } = this.props
 
-    if (loading) {
+    const mobile = __RUNTIME__.hints.mobile
```

Comment:
use withRuntimeContext

Reply:
This isn't included in the runtime context.

Reply:
ok :)